### ISSUE TITLE
[Feature] Use .json config file for arguments

### ISSUE THREAD

**Issue by m3yevn** [techlestial/gitlestial#3]
- [x] check inside project directory for gitlestial.config.json or .js to read configurations so that user won't be necessary to type in CLI for committing.

**Comment by m3yevn** [comment]
need to add a service which will read gitlestial.config.js (by default) if specify the specified path file, to get all arguments to perform operation

**Comment by PyaeHlianMoe** [comment]
**Default**
In default, config file will exist in the same directory as `LICENSE`. I will give the file name as `gitlestial-config.json`. Can execute by running `gitlestial run` command.

**User specified config file path**
In this scenario, user can specify their own config file path using this command. `gitlestial run --config <config_file path>`.

**gitlestial-config.json format**
```json
{
    "npmCommands": [
        "npm install",
        "npm link"
    ],
    "gitCommands": [
        "git add",
        "gitlestial commit-gen",
        "git push"
    ]
}
```

**Comment by m3yevn** [comment]
> **Default**
> In default, config file will exist in the same directory as `LICENSE`. I will give the file name as `gitlestial-config.json`. Can execute by running `gitlestial run` command.
> 
> **User specified config file path**
> In this scenario, user can specify their own config file path using this command. `gitlestial run --config <config_file path>`.
> 
> **gitlestial-config.json format**
> 
> ```json
> {
>     "npmCommands": [
>         "npm install",
>         "npm link"
>     ],
>     "gitCommands": [
>         "git add",
>         "gitlestial commit-gen",
>         "git push"
>     ]
> }
> ```

no, what i mean is gitlestial.config.js will contain all the arguments for git commit-gen or anything
can we make it as gitlestial.config.js to follow what usual cli tools name it e.g webpack.config.js 

can reference here https://gist.github.com/iamshaunjp/a1c8fb653ddd5d27c3e38ab55e3c0a44

example js config file
```
module.exports = {
 commitGen: {
 amount: 100,
 contributors: [example@example.com,example1@example.com],
 message: "just triggering commits",
}
}
```

**Comment by m3yevn** [comment]
the idea is to make user convenient by configuring the arguments like --amount 100 --contributors xx@123.com,xx@123.com -am "triggering commits"
